win32/ctags_vs2013.vcxproj.filters

```diff
@@ -818,4 +821,4 @@
       <Filter>Resource Files</Filter>
     </ResourceCompile>
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>

```

Comment:
Don't include this change in the pull request.
You can add parts to your commit with 'git commit -p'.